modules/3d-tiles/src/lib/ion/ion.js

```diff
@@ -45,11 +45,23 @@ export async function getIonAssets(accessToken) {
 // Return metadata for a specific asset assocated with token
 export async function getIonAssetMetadata(accessToken, assetId) {
   assert(accessToken, assetId);
-  const url = `${CESIUM_ION_URL}/${assetId}/endpoint`;
   const headers = {Authorization: `Bearer ${accessToken}`};
-  const response = await fetch(url, {headers});
+
+  const url = `${CESIUM_ION_URL}/${assetId}`;
```

Comment:
How about some comments about what we load from the different endpoint urls?

We could add links to the Cesium ION API documentation...